tests/test_io_utils.py

```diff
@@ -14,35 +15,61 @@
     "filename, mean, shape",
     [
         ("img.tif", 13577.494856, (2048, 2048, 3)),
-        ("img.jpg", 52.412425, (2048, 2048, 3)),
         ("img.png", 52.33736, (2048, 2048, 3)),
+        ("img.jpg", 52.412425, (2048, 2048, 3)),
         ("img-8bit.jp2", 47.09060, (343, 343, 3)),
         ("img-15bit.jp2", 0.3041897, (1830, 1830)),
         ("img-16bit.jp2", 0.3041897, (1830, 1830)),
     ],
 )
-def test_img_read(input_folder: str, output_folder: str, filename: str, mean: float, shape: Tuple[int, ...]) -> None:
-    is_jpeg = filename.endswith("jpg")
+def test_img_read(input_folder: str, filename: str, mean: float, shape: Tuple[int, ...]) -> None:
     img = read_data(os.path.join(input_folder, filename))
 
     assert img.shape == shape
+    assert np.mean(img) == approx(mean, abs=1e-4)
+    assert img.flags["WRITEABLE"], "Obtained numpy array is not writeable"
 
-    if is_jpeg or python_implementation() != "PyPy":
-        assert np.mean(img) == approx(mean, abs=1e-4)
 
-    assert img.flags["WRITEABLE"], "Obtained numpy array is not writeable"
+xml_root = ET.Element("EOPatch")
+xml_data = ET.SubElement(xml_root, "data")
+ET.SubElement(xml_data, "field1", name="BANDS-S2-L1C").text = "some value1"
+ET.SubElement(xml_data, "field2", name="CLP").text = "some value2"
+TEST_XML = ET.ElementTree(xml_root)
```

Comment:
It would be great to have this in a fixture, but as you probably noticed, fixtures and `parametrize` don't mix. UNLESS you use the [lazy-fixture](https://github.com/tvorog/pytest-lazy-fixture) plugin. We already use it in eo-learn and I see no reason not to use it here as well (it is just a develop dependency, so it doesn't bloat the package).

Please add it to the requirements and change the above test case into a fixture, as it should improve the readability.